src/mtev_conf.c

```diff
@@ -3739,7 +3760,11 @@ mtev_boolean mtev_conf_env_off(mtev_conf_section_t node, const char *attr) {
   return mtev_true;
 }
 
+static void safe_free_xpath(void *in) {
+  if(in) xmlXPathFreeContext(in);
+}
 void mtev_conf_init_globals(void) {
   mtev_conf_aco_recursion_counter_idx = aco_tls_assign_idx();
+  pthread_key_create(&xpath_ctxt_key, safe_free_xpath);
```

Comment:
I don't see where this is used. This seems to be the only reference to `xpath_ctxt_key` apart from the declaration.